Source/HTTP/WinHttp/winhttp_connection.cpp

```diff
@@ -1045,13 +1030,37 @@ void WinHttpConnection::callback_status_data_available(
     // Read new data into buffer
     if (newBytesAvailable > 0)
     {
-        pRequestContext->m_responseBuffer.resize(newBytesAvailable);
+        // Stage 1 performance optimization: For requests with known Content-Length,
```

Comment:
nit: "Stage 1" feels like a weird comment given I don't think I see any other "stages" of optimization here?

Reply:
Right, removed